docs/usage/types/strict_types.md

```diff
@@ -1,5 +1,8 @@
 Strict types enable you to prevent coercion from compatible types.
 
+## Strict types
+
+
 Pydantic provides the following strict types:
 
 - [`StrictBool`][pydantic.types.StrictBool]
```

Comment:
I feel like we should deprecate these. It seems much more composable to recommend`Strict[bool]`.

Reply:
@dmontagu thoughts?

Reply:
I guess I didn't look but I expected that to just work.

Reply:
I'd suggest extracting this idea into a new issue. Looks like a legitimate improvement that could be implemented in Pydantic V2.1+

Reply:
PYD-156